docs/synthetics/status-pages.md

```diff
@@ -0,0 +1,41 @@
+title: Status Pages
+description: Communicate the status of your services to your customers using status pages
+
+Using Sematext Synthetics you can create public status pages to share the status and response time of your services. Use the status pages to:
+
+* Communicate the status of your services to your customers
+* Share status updates to your customers during planned maintenance or outages
+* Create a dashboard of service status to share across internal teams
+
+## Create Status Page
+
+You can create multiple status pages under your account. Navigate to the `Synthetics -> Status Pages` screen to create a status page. 
+
+![Create Status Page](../images/synthetics/status-page-create.png)
+
+The following fields are required to create a status page:
+
+* **Monitors** - List of monitors to be included in the status page. All the active monitors from Apps for which you have OWNER or ADMIN role can be included in the status page. A monitor can be part of multiple status pages.
+* **Name** - Unique name for the status page.
+* **Header Name** - Company name to be displayed in the status page header.
+* **Header Logo** - URL of the company logo to be displayed in the status page header. An image with a height of 40px and white background is recommended.
+* **Response Time Aggregation** - Aggregation to apply for Response Time chart. The response time is aggregated in 1 hour buckets.
+* **Custom domain** - Optionally you can host the status page in your own custom domain.
+
+Every status page will have a unique public URL of the format - `https://apps.sematext.com/ui/statuspages/<unique-token>`. You can use this URL to preview the status page before opening to the public.
+
+Disabling the status page will block the public access for the status page.
+
+## Status Page UI
+
+![Status Page UI](../images/synthetics/status-page-ui.png)
```

Comment:
I would suggest replacing this with a link to our own public status page if/when we have one. This screenshot will most likely get out of date very soon.

Reply:
Yes. We can add a link to our own status page. But the screenshot is annotated with details of what each section means (show below).